dbms/src/Storages/Page/V3/PageDirectory.h

```diff
@@ -282,7 +287,14 @@ class VersionedPageEntries
         , create_ver(0)
         , delete_ver(0)
         , ori_page_id{}
-    {}
+    {
+        PageStorageMemorySummary::mem_sum_page_entries.fetch_add(sizeof(VersionedPageEntries<Trait>));
+    }
+
+    ~VersionedPageEntries()
+    {
+        PageStorageMemorySummary::mem_sum_page_entries.fetch_sub(sizeof(VersionedPageEntries<Trait>));
```

Comment:
The `sizeof()` may be very inaccurate, because there is a multi-map inside: `std::multimap<PageVersion, EntryOrDelete> entries`

Reply:
Yes, this summary is to cover "too many pages" scene, rather than "page of too many versions" scene.
AFAIK, there is no easy ways to record dynamic allocations by one class after it is constructed. So we can only do this far here. If we need a more accurate view, we can use some other trackers.

Reply:
We don't track VersionedPageEntries